doc/en/array.txt

```diff
@@ -119,21 +122,22 @@
    $ONELINER$
       Adjust the size of an array
    $SYNTAX$
-      ASize( <aArray>, <nLen> ) --> aTarget
+      ASize( <aArray>, <nLen> ) --> aArray
    $ARGUMENTS$
       <aArray> Name of array to be dynamically altered
-
-      <nLen> Numeric value representing the new size of <aArray>
+      
+      <nLen>  Numeric value representing the new size (i.e. number of elements)
+              of <aArray>
    $RETURNS$
-      <aTarget> an array pointer reference to <aTarget>.
+      The function returns a reference to <aArray>.
    $DESCRIPTION$
       This function will dynamically increase or decrease the size of
       <aArray> by adjusting the length of the array to <nLen> subscript
       positions.
-
-      If the length of the array <aArray> is shortened, those former
-      subscript positions are lost. If the length of the array is
-      lengthened a NIL value is assigned to the new subscript position.
+      
```

Comment:
Please config your editor to remove line ending spaces, like above (and few more places)